<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="total_passive">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total Passive&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cash_flow">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cash Flow&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="total_income">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total Income&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="total_expenses">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total Expenses&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyzeButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>500</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="QTreeView" name="income_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="expenses_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="assets_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>341</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="liabilities_view">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>341</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="opportunity_view">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>341</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
